spicepy/_http.py

```diff
@@ -0,0 +1,116 @@
+import datetime
+import json
+from typing import Any, Callable, Dict, NewType, Optional, Union
+from requests import Response, Session
+from requests.adapters import HTTPAdapter, Retry
+
+from .error import SpiceAIError
+
+# Literal not supported in python 3.7
```

Comment:
Might now be the time to minver 3.8?

Reply:
Yep, the reason we supported 3.7 was for Kaggle support - but I just checked and Kaggle now runs Python 3.10

Reply:
I'm going to move minimum version to 3.10 in the upcoming PR.

Reply:
Sorry yes, I meant 3.8